pipeline-runner/R/qc-2-filter_low_cellsize.R

```diff
@@ -19,7 +19,7 @@ filter_low_cellsize <- function(scdata, config, sample_id, task_name = 'cellSize
 
   # umi histogram plot
   numis <- sort(sample_subset$nCount_RNA, decreasing = TRUE)
-  plot1_data <- lapply(unname(numis), function(x) {
+  plot2_data <- lapply(unname(numis), function(x) {
```

Comment:
Change the designation of `plot_1` to `plot_2` and vice versa, to reflect the order of the plot (plot 1 on top and plot 2)